src/components/organisms/NavBarSchedule/NavBarSchedule.tsx

```diff
@@ -151,7 +155,13 @@ export const NavBarSchedule: React.FC<NavBarScheduleProps> = ({
       );
 
     const locatedEvents: LocationEvents[] = Object.entries(
-      groupBy(daysEvents, buildLocationString)
+      groupBy(
+        daysEvents.filter(
+          (event: { venueId: string }) =>
+            !event.venueId.match(REMOVE_EVENTS_FROM_VENUE)
+        ),
+        buildLocationString
+      )
```

Comment:
We can definitely rework this, but I think hiding all the events from a specific posterpage template might cause other problems — for example, there are other posterhalls than the main one, which have a limited number of events that should be shown. 
One alternative approach might be to add a parameter to the `event` type to `hideEvent?` so that if set to true, it doesn't show in the main schedule, but only in the personal schedule track. Would that be preferable?

Reply:
Regarding the change in `src/components/molecules/ScheduleEvent/ScheduleEvent.tsx`, I can revert that, but worth noting that if an event is very short, currently the title isn't displayed at all. The result of the change implemented here is that the title exceeds the length of the event box, but is visible, which I think has an advantage because it's possible to see the event title even for short events. I'll revert these changes, but wanted to mention that issue.

Reply:
For sure -- added here: https://github.com/sparkletown/sparkle/pull/1375/files#r651908164

Reply:
> > One alternative approach might be to add a parameter to the `event` type to `hideEvent`? so that if set to true, it doesn't show in the main schedule, but only in the personal schedule track.
> 
> The approach of marking events is more preferable for me. But why do we need to hide the events from the main schedule if they can appear in the personal schedule? I believe it'd confusing for the user if [s]he sees the events in the personal schedule but does not see the line of the venue/room where it's located.
> 
> one more note about this part of the code.
> 
> We filter all the posters to hide them from the schedule. But what if the poster is bookmarked? In your implementation, we see only its events in the "My Personal schedule line".
> But I believe we need to show the venue line in the schedule as well.
> 
> ![image](https://user-images.githubusercontent.com/63313732/121996914-79036500-cd77-11eb-8c8c-bb5824190f24.png)
> 

That's a good point — just a note of the issue to consider, which is that there around ~1400 posterpage venues, each of which has 3 events. I've imported those events into the ohbm platform so you can see the issue. The way the schedule is sorting, there are venues at the bottom of the list, after the 1400 posterpage venues, which might easily get lost. I know this is a very specific problem, but this is the one that the posterbookmarking was designed to address.

Reply:
I noticed that for all of these posters, they are being displayed with presumably their `venueId` rather than a name/human readable label? Is that by purpose/design? Why are we doing it that way?

![image](https://user-images.githubusercontent.com/753891/122154181-b9fa9700-cea7-11eb-839e-ebe2e5269b61.png)

Reply:
Thanks — I hadn't noticed that. The `venue.name` (and `venue/event/room`) is actually set to the `venue.id`, but that doesn't look nice at all. Thanks for the fresh pair of eyes — I'll update the `venue.name` and `room` in events accordingly.